ofl/heebo/METADATA.pb

```diff
@@ -14,9 +14,17 @@ fonts {
 }
 subsets: "hebrew"
 subsets: "latin"
+subsets: "latin-ext"
+subsets: "math"
 subsets: "menu"
+subsets: "symbols"
```

Comment:
@emmamarichal since the Latin is Roboto, it would worth to add the primary script for Hebrew so the font shows only Hebrew in the specimen page